tests/regression_tests/benchmark/test_model_performance.py

```diff
@@ -0,0 +1,62 @@
+import os
+from typing import List
+
+import pytest
+from expected_metric import ExpectedMetric
+
+from ludwig.benchmarking.benchmark import benchmark
+from ludwig.utils.data_utils import load_yaml
+
+
+def get_test_config_filenames() -> List[str]:
+    """Return list of the config filenames used for benchmarking."""
+    benchmark_directory = "/".join(__file__.split("/")[:-1] + ["configs"])
+    return [config_fp for config_fp in os.listdir(benchmark_directory)]
+
+
+def get_dataset_from_config_path(config_path: str) -> str:
+    """path/to/config/<dataset>.<descriptors>.yaml -> dataset."""
+    return os.path.basename(config_path).split(".")[0]
+
+
+@pytest.mark.benchmark
+@pytest.mark.parametrize("config_filename", get_test_config_filenames())
+def test_performance(config_filename, tmpdir):
+    benchmark_directory = "/".join(__file__.split("/")[:-1])
+    config_path = os.path.join(benchmark_directory, "configs", config_filename)
+    expected_test_statistics_fp = os.path.join(benchmark_directory, "expected_metrics", config_filename)
+    dataset_name = get_dataset_from_config_path(config_path)
+
+    if not os.path.exists(expected_test_statistics_fp):
+        raise FileNotFoundError(
+            """No corresponding expected metrics found for benchmarking config '{config_path}'.
+            Please add a new metrics YAML file '{expected_test_statistics_fp}'. Suggested content:
+
+            metrics:
+              - output_feature_name: <YOUR_OUTPUT_FEATURE e.g. SalePrice>
+                metric_name: <YOUR METRIC NAME e.g. accuracy>
+                expected_value: <A FLOAT VALUE>
+                tolerance_percent: 0.15"""
+        )
```

Comment:
Is there a way to automatically derive a reasonable suggested `ExpectedMetric` from a `BenchmarkingArtifact`?

Printing this out in a well-formatted way, alongside the "expected metric file is missing" error, would make it super clear and convenient to add a new expected metric file. WDYT?

Reply:
I wanted to check for the existence of this file before the training happens since the test will fail anyway if the expected metric YAML is not found.

The `BenchmarkingResult` object has a lot of attributes and we can definitely generate a message automatically, but this will require us to do the training first, then extract metrics from `test_statistics`. I think the current example should illustrates the YAML without giving specific values.

Reply:
Sounds good.

I anticipate that it will be useful to provide a separate utility (or extend an existing one) to run benchmarking config for a new benchmarking config and which prints out an `ExpectedMetric` yaml file that can be feasibly copy-pasted wholesale into a new file in the expected metrics dir.